### PR TITLE
add Class PersonalInfo with method

### PR DESCRIPTION
**Pull request by sergiipopil**
task 1 - add class PersonalInfo

### CODE REVIEW

**Comment by EugeneMz13** [comment]
Everything looks good

**Comment by MarkIlchenko** [comment]
Все гуд!

File: Classes/PersonalInfo.cs [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+        public PersonalInfo(string name, string surname) { 
+            _name= name;
+            _surname= surname;
+        }
+        public string GetUserInfo() {
+            var fullName = $"Full user name: {_name} {_surname}";
+            return fullName;
+        }
+    }
```
Comment (Bohdan103) [inline Classes/PersonalInfo.cs:22]: Looks fine!

**Review by MarkIlchenko — COMMENTED** [review]
Все гуд!

File: Classes/PersonalInfo.cs [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+using System.Threading.Tasks;
+
+namespace Task_PR_Team1.Classes
+{
+    internal class PersonalInfo
+    {
+        private string _name;
+        private string _surname;
+
+        public PersonalInfo(string name, string surname) {
```
Comment (Bohdan103) [inline Classes/PersonalInfo.cs:14]: Great

Comment (MarkIlchenko) [reply Classes/PersonalInfo.cs:14]: I'm agree

File: Program.cs [hunk 3]
```diff
@@ -1 +1,11 @@
...
+
+Console.Write("Enter your name: ");
+var name = Console.ReadLine();
+
+Console.Write("Enter your surname: ");
+var surname = Console.ReadLine();
+
+PersonalInfo personalInfo = new PersonalInfo(name, surname);
+var info = personalInfo.GetUserInfo();
+Console.WriteLine(info);
```
Comment (Bohdan103) [inline Program.cs]: All good!

**Review by Bohdan103 — APPROVED** [review]

**Review by MarkIlchenko — APPROVED** [review]
Good

**Pull request merged**